README.md

```diff
@@ -7,10 +7,12 @@ To experiment with this code, add some sample content and run `bundle exec jekyl
 
 ## Installation
 
+### Option 1: Install from GitHub
+
 Add this line to your Jekyll site's `Gemfile`:
 
 ```ruby
-gem "jekyll-clf-theme"
+gem "jekyll-clf-theme", git: "git@github.com:ubc-cpsc/jekyll-clf-theme"
```

Comment:
This shouldn't be needed if I release another one after the fix (I hope)

Reply:
This part I was assuming this is if you didn't clone https://github.com/ubc-cpsc/jekyll-clf-theme/ or have it installed on your machine etc. Otherwise wouldn't it need to be published at https://rubygems.org/ to write `gem "jekyll-clf-theme"` only?

(At least I couldn't get it to work without adding `git: "git@github.com:ubc-cpsc/jekyll-clf-theme"`)

Reply:
@natashamm this change is likely needed too, I just realized (moved our faculty template code over from bitbucket)

https://github.com/ubc-cpsc/jekyll-faculty-template/blob/main/Gemfile#L14